docs/Homebrew-Governance.md

```diff
@@ -78,20 +82,28 @@
 
 ## 6. Project Leader
 
-1. The Project Leader will represent Homebrew publicly, and manage all day-to-day technical decisions related to the operation of Homebrew.
+1. The Project Leader will represent Homebrew publicly, manage all day-to-day technical decisions, and resolve disputes related to the operation of Homebrew between maintainers, members, other contributors, and users.
 
 2. The Project Leader will be elected annually by a majority vote of Homebrew members. The PLC will nominate at least one candidate for Project Leader. Any member may nominate a candidate, or self-nominate. Nominations must be announced to the membership three weeks before the AGM.
 
 3. Any vacancy of the Project Leader will be filled by appointment of the PLC.
 
-4. After March 1, 2020, the Project Leader may not be a member of the PLC.
+4. The Project Leader may not be a member of the PLC.
+
+5. A technical decision of the Project Leader may be overruled by an ordinary resolution of the TSC.
+
+6. A non-technical decision of the Project Leader may be overruled by an ordinary resolution of the PLC.
+
+7. The Project Leader may be removed from the position by a special resolution of the membership.
 
 ## 7. Technical Steering Committee
 
-1. The TSC has the authority to decide on any technical matter, including technical disputes between Homebrew members. The TSC will also decide the long-term roadmap of Homebrew, including large features or major changes to the Homebrew software.
+1. The TSC has the authority to decide on any technical matter, including technical disputes between Homebrew members.
 
-2. The PLC will appoint between three and five members to the TSC. The Project Leader must be one of these appointees. Appointed TSC members will serve a term of one year or until the member's successor is appointed.
+2. The PLC will appoint between three and five members to the TSC. The Project Leader must be one of these appointees. After the 2021 AGM, PLC members should not be any of these appointees. Appointed TSC members will serve a term of one year or until the member's successor is appointed.
```

Comment:
I already commented on this before: Given the different tasks of the PLC and TSC, I don't see why one cannot serve on both. If we want to avoid the conflict of interest of PLC members appointing themselves to the TLC, I suggest:

```suggestion
2. The TSC consists of five members. Committee members are elected by majority vote of Homebrew members. Each TSC member will serve a term of one year or until the member's successor is elected. Any sudden vacancy in the TSC will be filled by the usual procedure for electing TSC members at the next general meeting, typically the next AGM.
```

Reply:
I think the idea is that the PLC and TSC should be empowered to disagree with each other. That gets kinda awkward if one is on both.

Reply:
The duties of the PLC and TSC don't overlap so I'm not sure in what scenario they would disagree with each other.

Reply:
In practice currently, perhaps. But the roles and responsibilities of the PLC and TSC aren't really clearly defined here, so there's no guarantee that they'd stay that way.

That said, I'm not sure non-overlapping duties necessarily preclude disagreement either. Think of, for instance, different branches of government, or, say, management vs the board of directors in a corporation.

Reply:
This still allows members to be in both groups. It just expresses a preference for this not to be the case. This is why it uses "should" instead of "must".
If there is a good reason for a member to occupy two seats in Homebrew's governance we can still pick that option.

Reply:
I feel like there should not be overlap between PLC and TSC because we have enough maintainers to avoid people needing to do both and they are pretty different roles.